resources/assets/js/views/Preview.vue

```diff
@@ -245,39 +244,39 @@ export default {
 
 		validateLayout(){
 
-			let $this = this;
+			let layoutName = `${this.currentLayout}-v${this.layoutVersion}`;
 
-			$this.layoutDefinition = $this.siteLayoutDefinitions[`${$this.currentLayout}-v${$this.layoutVersion}`];
+			this.layoutDefinition = this.siteLayoutDefinitions[layoutName];
 			// check that we have the same number of regions in our data as we have defined
-			if (Object.keys($this.pageData.blocks).length !== $this.layoutDefinition.regions.length) {
-				$this.layoutErrors.push('The regions on this page do not match the expected regions it should have.');
+			if (Object.keys(this.pageData.blocks).length !== this.layoutDefinition.regions.length) {
+				this.layoutErrors.push('The regions on this page do not match the expected regions it should have.');
 			}
 
 
-			_.each($this.layoutDefinition.regions, function (regionName) {
+			this.layoutDefinition.regions.forEach((regionDefinitionName) => {
 
 				// check that this defined region exist in the page's regions
-				if ($this.pageData.blocks[regionName] === void 0) {
-					$this.layoutErrors.push(`The region '${regionName}' was expected but not found on this page.`);
+				if (this.pageData.blocks[regionDefinitionName] === void 0) {
```

Comment:
Needs to be an else statement after this containing the code below it, as if pageData does not contain this region then there is no point seeing if the non-existant region contains the right sections.